### ISSUE TITLE
validate_content_type_of failing since 1.3.1

### ISSUE THREAD

**Issue by modosc** [igorkasyanchuk/active_storage_validations#292]
this seems like a duplicate of https://github.com/igorkasyanchuk/active_storage_validations/issues/280 except it's still not working with `1.3.3`.

in our model:
```ruby
  # BEGIN validations
  validates :file,
    presence: true,
    content_type: [
      "image/jpeg",
      "image/jpg",
      "image/png",
      "image/webp",
      "application/pdf",
    ],
    size: { less_than: 10.megabytes }
```

in our spec:
```ruby
    it {
      expect(subject).to validate_content_type_of(:file).allowing(
        "image/jpeg",
        "image/jpg",
        "image/png",
        "image/webp",
        "application/pdf",
      )
    }
```

the error:
```
  1) FileAttachment validations is expected to validate the content types allowed on :file
     Failure/Error: 
       expect(subject).to validate_content_type_of(:file).allowing(
         "image/jpeg",
         "image/jpg",
         "image/png",
         "image/webp",
         "application/pdf",
       )

       is expected to validate the content types of :file
         the following content types should be allowed: :image/jpeg, :image/jpg, :image/png, :image/webp, :application/pdf
         but :image/jpg was rejected
```

**Comment by kapso** [comment]
yea same issue

**Comment by nuxy** [comment]
Same issue here. See comment related to REGEX use in https://github.com/igorkasyanchuk/active_storage_validations/issues/282

**Comment by Mth0158** [comment]
Incoming bugfix release in the next hours/tomorrow to definitely fix this issue => we do not perform the extension vs content_type that causes this mess. Sorry for the unexpected breaking changes, it should have been tested more.

Let us know if you have any features idea to enhance the gem by the way :)

**Comment by Mth0158** [comment]
The fix has been released, upgrade to [1.3.4](https://github.com/igorkasyanchuk/active_storage_validations/releases/tag/1.3.4) to fix the issue :)
